### PR TITLE
[luxom] Add missing default translations

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.luxom/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
 			<bridge-type-ref id="bridge"/>
 		</supported-bridge-type-refs>
 		<label>switch</label>
```
Comment (jlaur) [inline bundles/org.openhab.binding.luxom/src/main/resources/OH-INF/thing/thing-types.xml:36]:
```suggestion
		<label>Switch</label>
```

**Review by jlaur — CHANGES_REQUESTED** [review]
`thing-type.luxom.switch.label` was inconsistent with the corresponding string in the XML, so needs to be corrected here.

File: bundles/org.openhab.binding.luxom/src/main/resources/OH-INF/thing/thing-types.xml (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.luxom/src/main/resources/OH-INF/thing/thing-types.xml:36]: You're right.

Comment (lolodomo) [reply bundles/org.openhab.binding.luxom/src/main/resources/OH-INF/thing/thing-types.xml]: Changed

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**